### PR TITLE
Add ConfirmClearCommand

### PR DESCRIPTION
**Pull request by Choonyan02**
## Description

Add ConfirmClearCommand

## Context

Addresses the need to have a secondary protection, in case the user accidentally enters clear

## Checklist

- [x] I have self-reviewed my changes.
- [x] I have added JavaDocs to all relevant methods.
- [x] I have updated the documentation: 
  - User Guide Table of Contents, Description and Summary.
  - Developer Guide, if applicable.
- [x] I have ran `./gradlew test` or `./gradlew check` and all checks pass.
- [x] I have updated my project portfolio with what I have contributed.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -65,6 +68,9 @@ public Command parseCommand(String userInput) throws ParseException {
         // Lower level log messages are used sparingly to minimize noise in the code.
         logger.fine("Command word: " + commandWord + "; Arguments: " + arguments);
 
+        isClearCommand = confirmClearCommand;
+        confirmClearCommand = false;
```
Comment (Vanessamae23) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:72]: should this start with is... since it is a boolean?

**Review by Vanessamae23 — CHANGES_REQUESTED** [review]
Looks good to me, just one comment for the coding standards

As much as possible, use a prefix such as is, has, was, etc. for boolean variable/method names so that linters can automatically verify that this style rule is being followed.

from the textbook website

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 1 above)
Comment (Choonyan02) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Okay, fixed!

**Review by Vanessamae23 — APPROVED** [review]
Okay great looks good to me now

**Pull request merged**